entsoe-cgmes-balances-adjustment/src/test/java/com/powsybl/entsoe/cgmes/balances_adjustment/util/CgmesVoltageLevelsAreaTest.java

```diff
@@ -25,14 +24,13 @@
  */
 class CgmesVoltageLevelsAreaTest {
 
-    private final String[] voltageLevelsIds = {"_d0486169-2205-40b2-895e-b672ecb9e5fc", "_4ba71b59-ee2f-450b-9f7d-cc2f1cc5e386", "_8bbd7e74-ae20-4dce-8780-c20f8e18c2e0", "_469df5f7-058f-4451-a998-57a48e8a56fe", "_b10b171b-3bc5-4849-bb1f-61ed9ea1ec7c"};
-    private final Network network = Network.read("controlArea.xiidm", getClass().getResourceAsStream("/controlArea.xiidm"));
-
     static final double DELTA_POWER = 1e-5;
 
     @Test
-    void testWithArea() {
-        CgmesControlArea cgmesArea = network.getExtension(CgmesControlAreas.class).getCgmesControlArea("_BECONTROLAREA");
+    void testWithAreaFromOldExtension() {
```

Comment:
Same thing with this class, can you create 2 methods for the 2 test cases and call them with both test files?